src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough6.java

```diff
@@ -0,0 +1,113 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThrough6 {
+    void foo(int i){
+        switch(i) {
+            case 1:
+                 int fallthru = 2; // fallthru
+            case 2: // ok
```

Comment:
It is not clear when you use "ok" .
If there is no good reason, let's remove all ok comments.
No comments means ok, so reader should look only at trailing comment as red flag

Reply:
doen